lib/OpenQA/Task/Asset/Download.pm

```diff
@@ -46,9 +61,15 @@ sub _download {
     my $log = $app->log;
     my $ctx = $log->context("[#$job_id]");
 
-    # bail if the dest file exists (in case multiple downloads of same ISO are scheduled)
-    return $ctx->info(my $msg = qq{Skipping download of "$url" to "$assetpath" because file already exists})
-      if -e $assetpath;
+    # skip download if the one dest file exists (in case multiple downloads of same ISO are scheduled)
+    my @existing_dest_files;
+    my @missing_dest_files;
+    -e $_ ? push @existing_dest_files, $_ : push @missing_dest_files, $_ for @$assetpaths;
+    if (@existing_dest_files) {
+        $ctx->info(my $msg = qq{Skipping download of "$url" because file "$existing_dest_files[0]" already exists});
+        _create_symlinks($job, $ctx, $existing_dest_files[0], \@missing_dest_files) if @missing_dest_files;
```

Comment:
I notice the log message is the same but it will create the symlinks now when the previous code would have just bailed out - maybe the message should be changed a bit, just to make it easier to spot in new logs

Reply:
Added debug logging to the symlink creation.